extension/src/format/parser.ts

```diff
@@ -364,3 +364,94 @@ export namespace LispParser {
         return result;
     }
 }
+
+
+export namespace LispParser {
+
+    interface ILspDocPair {
```

Comment:
can we make a subfolder somewhere, to hold this class, plus [the code started here](https://github.com/Autodesk-AutoCAD/AutoLispExt/pull/138/files#diff-30e8b6186a0e4537bb1f3855a033fc93fbbd1ed02b5907d99ce9a25a6525b975R45)?
it's not format anymore.

Reply:
Yes, I have been wanting to do that and this being something of an intermediate PR to break up some larger objective would make it a good one to take care of that.

Reply:
Parsing now has its own folder and everything parsing related is now in /src/parsing/